misc/server/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.2]
```

Comment:
```suggestion
## [0.12.3] - unreleased
```

Please bump the version in `misc/server/Cargo.toml` as well and run `cargo check` for `Cargo.lock` to be updated.